pom.xml

```diff
@@ -0,0 +1,124 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>io.quarkus</groupId>
+    <artifactId>quarkus-parent</artifactId>
+    <version>1.3.0.Final</version>
```

Comment:
I think that it is beneficial for clients of this extension, to use the BOM. The reason is that if we are using the BOM, we don't need to define the dependencies that are provided in the quarkus, for example:
https://github.com/datastax/cassandra-quarkus/pull/1/commits/a7edd3f2fc14adb55318b51322c618af29d78c13
Because we are using BOM, we don't need to manage those dependencies by ourselves. If there will be a new version of the quarkus, we would need only to increment this BOM version, and all dependant dependencies will be automatically updated.
Also, it will help us with keeping dependencies in sync with other extensions (please note that the majority of extensions are in the `quarkus` repo `extension` module and they do extend this BOM). wdyt?

Reply:
But precisely, our BOMs already include Quarkus BOM, see [here](https://github.com/datastax/cassandra-quarkus/blob/8b4c1149738c69e6a0c52be5934f16483b2c4799/poms/bom/pom.xml#L36-L42), which in turn inherit from `quarkus-parent`. So all Quarkus dependencies are correctly made available to users of these BOMs.
Our project itself does not need to inherit from that parent POM.
I think we can safely get rid of that inheritance. The only thing to add will be these two properties:

```
    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
    <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
```

Reply:
you right, the import of BOM is enough. Done

Reply:
yes, you are right. Import of the BOM should be enough. Done